src/sentry/models/release.py

```diff
@@ -270,6 +271,28 @@ def add_project(self, project):
         else:
             return True
 
+    def handle_commit_ranges(self, refs):
+        """
+        Takes commit refs of the form:
+        [
+            {
+                'previousCommit': None,
+                'commit': 'previous_commit..commit',
+            }
+        ]
+        """
+        def is_commit_range(commit):
+            return COMMIT_RANGE_DELIMITER in commit
+
+        def parse_commit_range(commit_range):
+            previous_commit, commit = commit_range.split(COMMIT_RANGE_DELIMITER)
```

Comment:
Is the order `previous_commit, commit` preferred? I assume so, but sometimes I see commits in reverse sorted order. So I want to double check.

Reply:
An reversed range wouldn't  work well with the repository integrations we have either. It is safe to assume the prev, current order you have now.